src/components/GlobeOptimized.jsx

```diff
@@ -395,20 +398,44 @@ export default function GlobeOptimized(props) {
       properties.forEach((property) => {
         switch (property) {
           case 'lat':
+            console.log('Rotation -->');
+            console.log(rotation);
             rotation[1] +=
```

Comment:
to simplify the logic here, we should verify the value first before assigning to `rotation[1]`, same applies to line 407 - 411